### PR TITLE
Canopy Cover Views

### PR DESCRIPTION
**Pull request by athai**
fixes issue #115

## Changes in this PR.

- [X] Write tests for edit_view and view_view
- [x] Create the necessary html templates
- [x] Reformat Canopy Cover model so that inline formsets can be used
- [x] Implement the views
- [x] Cleanup/update tests to reflect model changes

## Testing this PR.

**In Docker**
1. ``docker-compose run web bash``
2. ``./runtests.sh``

**In web browser**
1. ``docker-compose run web bash``
2. ``./get_sites.py``, ``./get_schools.py``
3. Visit ``localhost:8000/streamwebs/sites/<site-slug>/canopy/edit``
4. Fill out a data sheet and submit

### Expected Output.

```
[root@b1db827b0301 streamwebs]# ./runtests.sh 
[...]
Creating test database for alias 'default'...
......................................................................................................................................................................
----------------------------------------------------------------------
Ran 166 tests in 6.843s

OK
Destroying test database for alias 'default'...
```

Canopy Cover edit page
cc_edit_view

Canopy Cover view
cc_view_view

### CODE REVIEW

**Comment by alxngyn** [comment]
+1 looks as described

File: streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/riparian_transect_edit.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
                     <tr>
                         <th align='left'>{{ transect_form.slope.label }}</th>
                         <td>{{ transect_form.slope }}</td>
-                        <td>{{ transect_form.weather.errors }}</td>
+                        <td>{{ transect_form.slope.errors }}</td>
```
Comment (leian7) [inline streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/riparian_transect_edit.html]: thanks!!

File: streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/canopy_cover_view.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            <tr>
+                <th align='left'>{% trans "Site" %}</th>
+                <td>{{ site.site_name }}</td>
+            </tr>
+        </table>
+    </div>
+        <p>Placeholder for Canopy Cover graph</p>
+    </div>
+
+    <p><strong>Estimated Canopy Cover</strong>: {{ canopy_cover.est_canopy_cover }}/96</p>
```
Comment (leian7) [inline streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/canopy_cover_view.html]: Maybe calculate the percentage in the view first and then pass it directly to the template.

**Comment by leian7** [comment]
Can you add canopy cover links to site_detail.html? Also, I think it would be good to prevent the user from adding duplicate directions on the same form, i.e. not letting them submit four Easts. That can be done later though. +1 after links

**Pull request merged**